pipeline/terraform/main.tf

```diff
@@ -77,23 +77,23 @@ module "catalogue_pipeline_20201027" {
   # reindexer topic names.
 
   sierra_adapter_topic_arns = [
-    local.sierra_reindexer_topic_arn,
+    # local.sierra_reindexer_topic_arn,
     local.sierra_merged_bibs_topic_arn,
     local.sierra_merged_items_topic_arn,
   ]
 
   miro_adapter_topic_arns = [
-    local.miro_reindexer_topic_arn,
+    # local.miro_reindexer_topic_arn,
     local.miro_updates_topic_arn,
   ]
 
   mets_adapter_topic_arns = [
-    local.mets_reindexer_topic_arn,
+    # local.mets_reindexer_topic_arn,
     local.mets_adapter_topic_arn,
   ]
 
   calm_adapter_topic_arns = [
-    local.calm_reindexer_topic_arn,
+    # local.calm_reindexer_topic_arn,
```

Comment:
Didn't go you mean to do  this? Aren't we still reindexing?

Reply:
The messages are already on the queue, so was just doing this in case someone pulls up another pipeline. I haven't actually applied this part yet though.